app/src/main/java/com/globalaccelerex/mpossdksample/viewModel/ItemViewModel.kt

```diff
@@ -53,4 +56,9 @@ class ItemViewModel : ViewModel() {
         _itemsList.postValue(list)
 
     }
+
+    fun clearSession() {
+        _itemsList.value = loadCatalogListItems()
+        _totalPrice.value = BigDecimal.valueOf(0)
```

Comment:
You don't need to reset total price. Once you reset selected items, the transformation should run and take care of every other thing

Reply:
Has been resolved Sir